src/components/CIPipelineN/TaskTypeDetailComponent.tsx

```diff
@@ -240,26 +238,17 @@ export const TaskTypeDetailComponent = () => {
                     />
 
                     <div className="dc__position-rel">
-                        <CreatableSelect
-                            tabIndex={1}
+                        <SelectPicker
+                            inputId="containerImage"
                             value={selectedContainerImage}
                             options={containerImageOptions}
                             placeholder="Select container image or input value"
                             onChange={handleContainerImageSelector}
-                            styles={containerImageSelectStyles}
                             classNamePrefix="select"
-                            components={{
-                                MenuList: menuList,
-                                Option,
-                                IndicatorSeparator: null,
-                                ValueContainer,
-                            }}
```

Comment:
Pls do confirm if the menuList, Option and value container are not to be deleted

Reply:
Option and value container are common and using some other places as well

Reply:
cleaned menu list